src/features/chat/components/ChatList/index.tsx

```diff
@@ -0,0 +1,37 @@
+import VerifyIcon from '@/libs/assets/VerifyIcon';
+import * as s from './style.css';
+import { Link } from 'react-router';
+
+// TODO: 나중에 수정하기~~~
+interface Props {
+  nickName: string;
+  message: string;
+  count: number;
+  chatRoomId: number;
+  isVerified: boolean;
+}
+
+const ChatList = ({ nickName, message, count, chatRoomId, isVerified }: Props) => {
+  return (
+    <Link className={s.List} to={`/chatRoom/${chatRoomId}`}>
```

Comment:
```suggestion
    <Link className={s.List} to={`/chatroom/${chatRoomId}`}>
```

라우터에는 이렇게 되어있던데 상관 없나??
url에는 대문자 잘 안쓰니까 만약에 chatRoom으로 쓰고 싶은거면 chat-room으로 해주는 것도 괜찮아보여요

Reply:
chatroom으로 변경했슴다